tests/constraints.in

```diff
@@ -0,0 +1,5 @@
+# Known limitations for indirect/transitive dependencies.
+resolvelib < 0.9.0
+sphinx == 4.2.0
+rstcheck == 3.3.1
+antsibull-docs == 1.11.1
```

Comment:
This looks like a too new version. stable-2.14 has 1.7.0, so stable-2.13 should have something older.

Reply:
In ansible/ansible's stable-2.13 it was 1.0.0 before the docs build requirements were deleted in https://github.com/ansible/ansible/commit/abab52a7b411128475aeea3adde84ba32e1fdd55.

Reply:
Thanks @felixfontein gosh I thought I updated the version to match that. Just tried it locally and 1.0.0 seems to be fine.